randovania/games/fusion/logic_database/Sector 5 ARC.txt

```diff
@@ -819,7 +817,8 @@ Extra - room_id: [24]
   > Door to Kago Speedway
       Space Jump
   > Door to Groznyj Grad
-      Can Use Power Bombs
+      # Screw into kago to reach hole
+      Screw Attack or Can Use Power Bombs
```

Comment:
how are you supposed to reach the hole with screw only, when the hole is blocked by a PB block